src/Symfony/Component/Process/ProcessBuilder.php

```diff
@@ -226,6 +226,20 @@ public function setOption($name, $value)
     }
 
     /**
+     * Sets proc_open options.
+     *
+     * @param array $options Key value pair of options
+     *
+     * @return ProcessBuilder
+     */
+    public function setOptions(array $options)
```

Comment:
@Modelizer we use `set` when overwriting, `add` when adding. In a singular form, there's no difference. There is a difference, however, between `addOptions` and `setOptions`. First adds options and doesn't touch any options that were already there. While the later overwrites all existing options with new ones.